src/server/plugins/engine/pageControllers/index.ts

```diff
@@ -1,6 +1,9 @@
 export { HomePageController } from '~/src/server/plugins/engine/pageControllers/HomePageController.js'
-export { PageController } from '~/src/server/plugins/engine/pageControllers/PageController.js'
 export { StartPageController } from '~/src/server/plugins/engine/pageControllers/StartPageController.js'
+export {
+  QuestionPageController, // Export alongside alias
+  QuestionPageController as PageController
+} from '~/src/server/plugins/engine/pageControllers/QuestionPageController.js'
```

Comment:
For compatibility, temporarily support `PageController` as an alias of `QuestionPageController`